README.md

```diff
@@ -43,6 +43,18 @@ docker run -v /path/to/godata:/godata -v /path/to/home-dir:/home/go gocd/gocd-se
 
 > **Note:** Ensure that `/path/to/home-dir` and `/path/to/godata` is accessible by the `go` user in container (`go` user - uid 1000).
 
+## Running custom entrypoint scripts
+
+To execute custom scripts during the container boostrap, but **before** the gocd server starts just do
+
+add `-v /path/to/your/script.sh:/docker-entrypoint.d/your-script.sh ` so
```

Comment:
I'd probably also clarify that the entire `docker-entrypoint.d` can be volume mounted if desired.